src/TestEngine/TestUtils/TestConsensusContext.cs

```diff
@@ -0,0 +1,40 @@
+using Neo.Consensus;
+using Neo.Cryptography.ECC;
+using Neo.Network.P2P.Payloads;
+using Neo.Persistence;
+using Neo.SmartContract;
+using Neo.Wallets;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Neo.TestingEngine
+{
+    public class TestConsensusContext : ConsensusContext
+    {
+        public TestConsensusContext(ECPoint[] validators, Wallet wallet, IStore store) : base(wallet, store)
+        {
+            this.Validators = validators;
+            this.CommitPayloads = new ConsensusPayload[validators.Length];
+        }
+
+        public void SetBlock(Block block)
+        {
+            this.Block = block;
+        }
+
+        public Block CreateBlock()
```

Comment:
@lock9 could you give me write rights? 
```suggestion
        public new Block CreateBlock()
```